modules/claims_api/lib/claims_api/v2/disability_compensation_evss_mapper.rb

```diff
@@ -88,16 +78,6 @@ def veteran_meta
         @evss_claim[:veteran][:emailAddress] = @data.dig(:veteranIdentification, :emailAddress, :email)
         @evss_claim[:veteran][:fileNumber] = @file_number
       end
-
-      # Convert 12-05-1984 to 1984-12-05 for Docker container
-      def format_service_periods(service_period_dates)
-        service_period_dates.each do |sp_date|
-          begin_year = Date.strptime(sp_date[:activeDutyBeginDate], '%m-%d-%Y')
-          end_year = Date.strptime(sp_date[:activeDutyEndDate], '%m-%d-%Y')
-          sp_date[:activeDutyBeginDate] = begin_year.strftime('%Y-%m-%d')
-          sp_date[:activeDutyEndDate] = end_year.strftime('%Y-%m-%d')
-        end
-      end
     end
```

Comment:
Removed per this convo: https://oddball.slack.com/archives/C019XKGS1CN/p1695844255845029